lib/utils.js

```diff
@@ -50,7 +50,7 @@ exports.validateArg = function validateArg (args, config) {
         break;
 
       case Function:
-        if (toString.call(value) !== '[object Function]') {
+        if (typeof obj !== "undefined" && toString.call(value) !== '[object Function]') {
```

Comment:
$ grep -r validateArg *
lib/memcached.js:    if (query.validate && !Utils.validateArg(query, this)) {
lib/utils.js:exports.validateArg = function validateArg (args, config) {

"system wide" seems to be this one call, so in all cases that I'm aware of, it is the callback.  So in all of those cases, "undefined" is actually a valid value for that argument.